src/components/pages/home/cta/cta.jsx

```diff
@@ -79,68 +79,70 @@ const CTA = () => {
   }, [isInView, controls]);
 
   return (
-    <motion.section
-      className="safe-paddings bg-black pt-[504px] text-center 3xl:pt-[397px] 2xl:pt-[344px] xl:pt-[310px] lg:pt-[238px] md:pt-[160px]"
-      initial="from"
-      animate={controls}
-    >
-      <Container className="z-20" size="md" ref={animationVisibilityRef}>
-        <Heading id="cta-title" tag="h2" size="xl" theme="white" ref={titleRef}>
-          <BlinkingText
-            text="Made for developers"
-            parentElement={titleRef.current}
-            shouldAnimationStart={isInView}
-          />
-        </Heading>
-        <motion.p className="t-3xl mt-5 text-white 2xl:mt-4" variants={pVariants}>
-          Launch serverless Postgres with a single command
-        </motion.p>
-        <motion.div
-          id="cta-input"
-          variants={inputWrapperVariants}
-          className="relative mx-auto mt-8 max-w-[834px] 3xl:max-w-[738px] 2xl:mt-7 2xl:max-w-[610px] xl:mt-6 xl:max-w-[498px] lg:max-w-[584px]"
-        >
-          <motion.div
-            id="cta-input-background"
-            className="absolute -bottom-3.5 -left-3.5 h-full w-full rounded-full 2xl:-bottom-2.5 2xl:-left-2.5 xl:-bottom-2 xl:-left-2 md:w-[calc(100%+8px)]"
-            variants={inputPanelVariants}
-            aria-hidden
-          />
-          <div className="relative flex items-center justify-between rounded-full border-4 border-black bg-white p-2 pl-9 2xl:p-1.5 2xl:pl-7 xl:p-1 xl:pl-6 md:justify-center md:px-0 md:py-[22px]">
-            <span className="t-3xl whitespace-nowrap font-mono font-bold !leading-none">
-              $ psql -h pg.neon.tech
-            </span>
-            <Button
-              className="relative md:hidden"
-              size="sm"
-              theme="secondary"
-              onClick={handleButtonClick}
-            >
-              <span className={clsx({ 'opacity-0': isCopied })}>Copy</span>
-              <span
-                className={clsx(
-                  'absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2 opacity-0',
-                  { 'opacity-100': isCopied }
-                )}
-              >
-                Copied!
+    <LazyMotion features={domAnimation}>
```

Comment:
As you need this `<LazyMotion features={domAnimation}>` to wrap some components. You may consider a custom wrapper and just wrap on Invocation place if that helps to make changes easy.

Reply:
Thanks for the suggestion. I kept `<LazyMotion features={domAnimation}>` explicit for clarity. Introducing a wrapper will save you one less import. Let's discuss if you feel strongly about it.